### PR TITLE
Get repo building again

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

File: src/Benchmarks.Utility/Helpers/Runtimes.cs [hunk 1]
```diff
@@ -9,31 +9,8 @@ namespace Benchmarks.Utility.Helpers
...
-                return RuntimeFlavor.CoreClr;
-            }
-
-            // Remaining possibility is Mono and that's not currently supported.
-            Assert.False(true, $"Unknown framework {runtimeType}");
-
-            return RuntimeFlavor.Clr;
-        }
-
         public static string GetFrameworkName(string runtimeType)
```
Comment (dougbu) [inline src/Benchmarks.Utility/Helpers/Runtimes.cs]: Left these methods here primarily to limit appearances of the string `"netcoreapp2.0"`.

File: src/Benchmarks.Framework/Benchmarks.Framework.csproj [hunk 2]
```diff
@@ -3,16 +3,15 @@
...
+    <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Testing" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="$(AspNetCoreVersion)" />
-    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="$(PlatformAbstractionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="$(AspNetCoreVersion)" />
```
Comment (BrennanConroy) [inline src/Benchmarks.Framework/Benchmarks.Framework.csproj]: Can we remove this dependency? Its been obsoleted

File: test/TestApp.Test/TestApp.Test.csproj [hunk 3]
```diff
@@ -30,7 +27,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Testing" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(AspNetCoreVersion)" />
-    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="$(PlatformAbstractionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="$(AspNetCoreVersion)" />
```
Comment (BrennanConroy) [inline test/TestApp.Test/TestApp.Test.csproj]: Same, remove?

File: src/Benchmarks.Framework/Benchmarks.Framework.csproj (hunk 2 above)
Comment (dougbu) [inline src/Benchmarks.Framework/Benchmarks.Framework.csproj]: Huh? InternalAbstactions was obsoleted and we moved to this.

Comment (BrennanConroy) [inline src/Benchmarks.Framework/Benchmarks.Framework.csproj]: https://github.com/aspnet/PlatformAbstractions

**Comment by dougbu** [comment]
:up::date:

**Comment by dougbu** [comment]
ce1acc8

**Pull request closed without merging**